tests/network/vmi_istio.go

```diff
@@ -207,7 +208,7 @@ var istioTests = func(vmType VmType) {
 				Expect(err).NotTo(HaveOccurred())
 				Eventually(func() error {
 					return migrationCompleted(migration)
-				}, tests.MigrationWaitTime, time.Second).Should(Succeed(), fmt.Sprintf(" migration should succeed"))
+				}, libmigration.MigrationWaitTime, time.Second).Should(Succeed(), fmt.Sprintf(" migration should succeed"))
```

Comment:
nit: sounds like it has similarities in `libmigration`? Maybe extract a func and move it or see whether we can use something from over there?

Reply:
I don't have to extract anything, since the function is already present: `RunMigrationAndExpectToComplete`.
Thank you